custom-namespaces.yaml

```diff
@@ -873,6 +873,18 @@ firefox_desktop:
       type: ping_view
       tables:
         - table: mozdata.default_browser_agent.default_browser
+    clients_first_seen_28_days_later:
+      type: table_view
+      tables:
+        - table: mozdata.telemetry.clients_first_seen_28_days_later
```

Comment:
@irrationalagent I think clients_first_seen_28_days_later was added by @ANich in this PR https://github.com/mozilla/lookml-generator/pull/821. The explore has been extended in spoke-default here: https://github.com/mozilla/looker-spoke-default/pull/668 and can be found under the firefox desktop namespace as desktop acquisition funnel.